exec/java-exec/src/main/java/org/apache/drill/exec/server/options/OptionValue.java

```diff
@@ -146,6 +164,40 @@ public static OptionValue create(AccessibleScopes type, String name, Object val,
     throw new IllegalArgumentException(String.format("Unsupported type %s", val.getClass()));
   }
 
+  private static OptionValue fromString(AccessibleScopes type, String name,
+      String val, OptionScope scope, Kind kind) {
+    val = val.trim();
```

Comment:
We don't expect value to be null? Correct?

Reply:
No, we don't. However, added an argument check to enforce non-null.